### PR TITLE
Add unit tests for db/repos/delegates - Closes #1567

### LINKED ISSUE #1567: Add unit test coverage for db/repos/delegates
Parent: #1484

### Expected behavior
We need 100% coverage for db/repos/delegates

### Actual behavior
Currently there is no coverage for this repo.

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?
No unit test coverage for db/repos/delegates

### How did I fix it?
Added test cases thoroughly for every method in db/repos/delegates. And got 100% coverage for that file.

### How to test it?
```
npx mocha test/unit/db/repos/delegates.js
```

### Review checklist

* The PR solves #1567 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: db/sql/delegates/get_delegates_by_public_keys.sql [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 SELECT encode("publicKey", 'hex') AS "publicKey",
        username,
-       address
```
Comment (4miners) [inline db/sql/delegates/get_delegates_by_public_keys.sql]: That query don't need order by

File: test/unit/db/repos/delegates.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+				).to.be.eql(fork.delegatePublicKey);
+				expect(result[0].blockId).to.be.eql(fork.blockId);
+				expect(result[0].blockHeight).to.be.eql(fork.blockHeight);
+				expect(result[0].previousBlock).to.be.eql(fork.previousBlock);
+				expect(result[0].blockTimestamp).to.be.eql(fork.blockTimestamp);
+				return expect(result[0].cause).to.be.eql(fork.cause);
+			});
+
+			const fork = forksFixtures.Fork();
+			Object.keys(fork).forEach(attr => {
```
Comment (4miners) [inline test/unit/db/repos/delegates.js]: Something is wrong with structure here - `forEach` with `expect` should be inside `it`, not `it` inside `forEach`

File: test/unit/db/repos/delegates.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+					'address',
+					'vote'
+				);
+				expect(result.map(r => r.publicKey)).to.include(account1.publicKey);
+				return expect(result.map(r => r.publicKey)).to.include(
+					account2.publicKey
+				);
+			});
+
+			it('should return list sorted list of delegates by asc vote and desc public key ', function*() {
```
Comment (4miners) [inline test/unit/db/repos/delegates.js]: remove that test after remove order by from query

**Comment by nazarhussain** [comment]
@karmacoma There is only one test case remaining to implement. Which will be done as soon the `dapps` tests PR https://github.com/LiskHQ/lisk/pull/1607 will be merged. 

https://github.com/LiskHQ/lisk/pull/1614/files#diff-9fa2ac8b0b056e5bd9225b0a8e4f418dR194

**Pull request merged**